Hitting  TypeError while creating "default_buckets_backingstore" due to incorrect comparision

Hitting  TypeError while creating "default_buckets_backingstore" due to incorrect comparision
RP link: https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/678/23075/1110916/1110999/log
Error snippet:
```
tests/functional/object/mcg/test_default_backingstore_override.py:59: 
tests/conftest.py:7397: in _override_nb_default_backingstore_implementation
    alt_bs_name = backingstore_factory("oc", bs_dict)[0].name
ocs_ci/ocs/resources/backingstore.py:393: in _create_backingstore
    mcg_obj.check_backingstore_state(
ocs_ci/ocs/resources/mcg.py:828: in check_backingstore_state
    for reached_state in TimeoutSampler(timeout, 10, _check_state):

self = <ocs_ci.utility.utils.TimeoutSampler object at 0x7f0a66fc6130>
timeout = None, sleep = 10
func = <function MCG.check_backingstore_state.._check_state at 0x7f0a677474c0>
func_args = (), func_kwargs = {}

def __init__(self, timeout, sleep, func, *func_args, **func_kwargs):
    self.timeout = timeout
    self.sleep = sleep
    # check that given timeout and sleep values makes sense

  if self.timeout < self.sleep:
E       TypeError: '<' not supported between instances of 'NoneType' and 'int'


ocs_ci/utility/utils.py:1405: TypeError
```

--- a/ocs_ci/ocs/resources/backingstore.py
+++ b/ocs_ci/ocs/resources/backingstore.py
@@ -284,7 +284,7 @@ def backingstore_factory(request, cld_mgr, mcg_obj, cloud_uls_factory):
         },
     }
 
-    def _create_backingstore(method, uls_dict, timeout=None):
+    def _create_backingstore(method, uls_dict, timeout=600):
         """
         Tracks creation and cleanup of all the backing stores that were created in the scope
 

